### PR TITLE
For #252: Add License information to README

### PR DESCRIPTION
**Pull request by shresths**
Addresses one point of #252, for License information on the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ Contributions are always welcome, no matter how large or small. Before contribut
 please read the [code of conduct](code-of-conduct.md).
 
 See [Contributing](CONTRIBUTING.md).
+
+## License
+
+Copyright (c) 2016 The OpenTracing Authors.
```
Comment (yurishkuro) [inline README.md]: `2016-present` would be more correct.

**Comment by reimertz** [comment]
Do we really want to add a copyright to this file? Doesn’t that contradict our Apache license?

**Comment by yurishkuro** [comment]
how does it contradict the license? The license is a permission to use the materials. Copyright exists even without declaring it, but stating it explicitly establishes who exactly it belongs to.

In the code repositories we always ask people to include headers with the source files that does the same ([example][1]), but since there are no headers in the doc files, having it here in the readme makes sense.

[1]: https://github.com/opentracing/opentracing-java/blob/master/opentracing-api/src/main/java/io/opentracing/Scope.java#L2

File: README.md (hunk 1 above)
Comment (shresth27) [reply README.md]: @yurishkuro Done! Please review. Thanks. :)

**Pull request merged**